src/index.tsx

```diff
@@ -0,0 +1,559 @@
+import React from 'react'
```

Comment:
I don't have too much to say about this migration, it's really great! I just have a preference for decomposing the React variable to get a bit more concise code (e.g `forwardRef` instead of `React.forwardRef`)

Reply:
I prefer the React.* notation, so I can differentiate React and project code, but is only personal taste. I'll be decomposing for next commit.